app/main.py

```diff
@@ -1,8 +1,40 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str)\
+            -> None:
```

Comment:
Bad indentation, don't use `\` in this case. If you have too many arguments -> start them from a new line as you did in `CarWasStation` `init` method